lib/kdump_utils.pm

```diff
@@ -533,4 +543,33 @@ sub full_kdump_check {
     }
 }
 
+=head2 set_kdump_config
+
+ set_kdump_config($option, $value);
+
+This function modifies a configuration option within the F</etc/sysconfig/kdump> file.
+
+=over 4
+
+=item B<$option>
+
+Name of the configuration option in the kdump configuration file.
+
+=item B<$value>
+
+Value for the configuration option.
+
+=back
+
+=cut
+
```

Comment:
ideally we move all to pod's style comments, but definitely a matter for a later PR